avocado/utils/multipath.py

```diff
@@ -193,9 +192,11 @@ def is_failed():
 
 def reinstate_path(path):
     """
-    reinstating the individual paths
-    :param disk_path: disk path. Example: sda, sdb.
-    :return: True or False
+    Reinstate the individual path
+    Reinstate the individual paths
```

Comment:
There's a (mostly) duplicate line here

Reply:
I had to rebase and fix issues, left this. good catch!